apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -0,0 +1,208 @@
+				width: 32px;
+			}
+			.copy {
+				line-height: 1.4;
+				margin: 0;
+			}
+			.marketing-buttons .button {
+				background: #fff;
+				border-radius: 2px;
+				border: 1px solid #fff;
+				box-sizing: border-box;
+				color: #117ac9;
+				display: block;
+				font-size: 16px;
+				font-weight: 700;
+				line-height: 21px;
+				padding: 13px;
+				text-align: center;
+				text-overflow: ellipsis;
+				text-decoration: none;
+				transition: opacity .15s ease-out;
+				white-space: nowrap;
+				width: 100%;
+			}
+			.marketing-buttons .button-secondary,
+			.marketing-buttons .button-secondary:hover,
+			.marketing-buttons .button-secondary:focus {
+				background: transparent;
+				color: #fff;
+			}
+			.marketing-buttons .button:hover,
+			.marketing-buttons .button:focus {
+				opacity: .85;
+			}
+			@media screen and ( min-width: 660px ) {
+				.description,
+				.copy {
+					font-family: Recoleta, Georgia, "Times New Roman", Times, serif;
+				}
+				.name {
+					font-size: 23px;
+				}
+				.description {
+					font-size: 69px;
+				}
+				.marketing {
+					align-items: center;
+					display: flex;
+					justify-content: space-between;
+					padding-bottom: 24px;
+				}
+				.marketing-copy {
+					margin-right: 16px;
+				}
+				.marketing-buttons {
+					display: flex;
+				}
+				.marketing-buttons p {
+					margin: 0;
+				}
+				.marketing-buttons p:nth-child(2) {
+					margin-left: 8px;
+				}
+				.marketing-buttons .button {
+					font-size: 13px;
+					padding: 7px 13px;
+					min-width: 145px;
+				}
+			}
+			@media screen and ( min-width: 661px ) {
+				p, ul, ol {
+					font-size: 16px;
+					letter-spacing: 0;
+				}
+			}
+			@media screen and ( min-width: 960px ) {
+				.name {
+					font-size: 28px;
+					margin-bottom: 16px;
+				}
+				.description {
+					font-size: 99px;
+				}
+				.copy {
+					font-size: 19px;
+				}
+			}
+			@media screen and ( min-width: 1040px ) {
+				.body {
+					-ms-grid-columns: (1fr)[12];
+					grid-template-columns: repeat(12, 1fr);
+				}
+				.inner {
+					-ms-grid-column: 2;
+					grid-column-start: 2;
+					-ms-grid-column-span: 10;
+					grid-column-end: span 10;
+				}
+				.marketing {
+					padding-bottom: 32px;
+				}
+			}
+		</style>
+	</head>
+	<body class="body">
+		<div class="inner">
+			<div class="main">
+				<div class="name"><?php echo esc_html( get_bloginfo( 'name' ) ); ?></div>
+				<div class="description"><?php esc_html_e( 'Coming Soon', 'full-site-editing' ); ?></div>
+				<p>(v2 — remove this line when finished testing)</p>
+			</div>
+			<div class="marketing">
+				<?php if ( ! is_user_logged_in() ) : ?>
+					<div class="marketing-copy">
+						<img src="/wp-content/themes/a8c/domain-landing-page/wpcom-wmark-white.svg" alt="WordPress.com" class="logo" />
+						<p class="copy"><?php echo esc_html( fix_widows( __( 'Build a website. Sell your stuff. Write a blog. And so much more.', 'full-site-editing' ), array( 'mobile_enable' => true ) ) ); ?></p>
```

Comment:
Not sure if this is in the scope of this PR, but will assumed WordPress.com globals such as these work in Atomic? 

`fix_widows` exists on WordPress.com as a blog-plugin but not is undefined in wp-env 

<img width="1045" alt="Screen Shot 2020-09-09 at 1 23 37 pm" src="https://user-images.githubusercontent.com/6458278/92550777-afb7c680-f29f-11ea-87fd-f4f25ae943e7.png">

Same for `$login_url` and `localized_wpcom_url()` below.

Reply:
Maybe we should wait to see wether we're going to redesign the fallback coming soon page before worrying about fixing widows.

Missing `$login_url` and `localized_wpcom_url()` is interesting! I wonder if we should just the hide the banner if they're missing? Maybe another design/product question. It's kind of like the "Powered by WordPress.com" links not being on higher plans. Maybe if they're on Atomic then we can hide the WordPress.com banner because they have a plan?

Reply:
Is it risk free now to deploy this at Atomic? I.e. this code won't be run?

Reply:
It can't run unless `WPCOM_PUBLIC_COMING_SOON` is defined, so it'll only run if devs manually set that constant on their Atomic site to try it out.